src/Mvc/Mvc.NewtonsoftJson/test/NewtonsoftJsonInputFormatterTest.cs

```diff
@@ -418,5 +481,26 @@ private sealed class UserLogin
             [JsonProperty(Required = Required.Always)]
             public string Password { get; set; }
         }
+
+        public class TypeWithPrimitives
+        {
+            public DateTime DateValue { get; set; }
```

Comment:
To confirm: Json.NET has a converter that may allow a `FormatException` to escape?

Reply:
Yup, the `IsoFormatter` one. Think that's in use here.